resources/var/tmp/resources/init.groovy.d/scripts/init030CRITinstallplugins.groovy

```diff
@@ -91,50 +72,50 @@ def plugins = [
         'pipeline-github-lib',
         'authorize-project',
         'pipeline-stage-view'
-];
+]
 
-def additionalPluginPath = "config/jenkins/additional.plugins";
+def additionalPluginPath = "additional.plugins"
 
-if (keyExists(additionalPluginPath)) {
-    println("Install additional plugins");
-    def additionalPluginList = getValueFromEtcd(additionalPluginPath);
-    def additionalPlugins = additionalPluginList.split(',');
+if (ecoSystem.keyExists("dogu", additionalPluginPath)) {
+    println("Install additional plugins")
+    def additionalPluginList = ecoSystem.getDoguConfig(additionalPluginPath)
+    def additionalPlugins = additionalPluginList.split(',')
     for (additionalPlugin in additionalPlugins) {
         println("Add Plugin " + additionalPlugin)
         plugins.add(additionalPlugin)
     }
 } else {
-    println("No additional plugins configured");
+    println("No additional plugins configured")
 }
 
 // add sonar plugin to Jenkins if SonarQube is installed
-if (keyExists("dogu/sonar/current")) {
-    plugins.add('sonar');
+if (ecoSystem.isInstalled("sonar")) {
+    plugins.add('sonar')
```

Comment:
does it work? I thought in MN dogu.json from other Dogus wouldn't be mounted

Reply:
Added optional dependencies to dogu.json